README.md

```diff
@@ -13,9 +13,9 @@
 #import "lib.typ" as cmarker
 #let markdown = read("README.md")
 #cmarker.render(
-	markdown.slice(markdown.position("</picture>") + "</picture>".len()),
-	h1-level: 0,
-	smart-punctuation: false,
+  markdown.slice(markdown.position("</picture>") + "</picture>".len()),
```

Comment:
I’m not super opposed to this formatting change, but since it affects so many lines, I don’t think it should happen in this PR.

Maybe it was done accidentally – do you know how that could’ve happened?

Reply:
Sorry. Its a mistake. My editor changes tabs to space. I forgot to only stage the specific hunk and instead commit the entire file. Just a sec